assignments/callbacks.js

```diff
@@ -38,28 +38,59 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
   console.log(test2); // "this Pencil is worth a million dollars!"
 */
 
+// function getLength(items, callback) {
+//   return callback(items);
+// }
 
-function getLength(arr, cb) {
-  // getLength passes the length of the array into the callback.
-}
+// const callback = items => {
+//   return items.length;
+// };
 
-function last(arr, cb) {
-  // last passes the last item of the array into the callback.
-}
+// console.log(getLength(items, callback));
 
-function sumNums(x, y, cb) {
-  // sumNums adds two numbers (x, y) and passes the result to the callback.
-}
+// function last(arr, callback) {
+//   // last passes the last item of the array into the callback.
+//   return callback(arr[arr.length-1]);
+// }
 
-function multiplyNums(x, y, cb) {
-  // multiplyNums multiplies two numbers and passes the result to the callback.
-}
+// const callback = function callback(items) {
+//   return items
+// }
+
+// console.log(last(items, callback));
```

Comment:
Make sure, if you changed the name for your second callback function, you want to change how it's passed in the this original function as well:
```
console.log(last(items, callback2));
```